web_src/js/features/repo-legacy.js

```diff
@@ -607,15 +607,14 @@ function initRepoIssueCommentEdit() {
   $(document).on('click', '.edit-content', onEditContent);
 
   // Quote reply
-  $(document).on('click', '.quote-reply', function (event) {
+  $(document).on('click', '.quote-reply', async function (event) {
```

Comment:
If you really need this `async`, the `event.preventDefault()` should be put before any await, it's better to move it to the beginning of the function.

https://docs.gitea.io/en-us/guidelines-frontend/#async-functions

> It’s not recommended to use async event listeners, which may lead to problems. The reason is that the code after await is executed outside the event dispatch. Reference: https://github.com/github/eslint-plugin-github/blob/main/docs/rules/async-preventdefault.md